src/ext/php7/serializer.c

```diff
@@ -222,8 +221,8 @@ static void _serialize_stack_trace(zval *meta, zval *trace) {
     add_assoc_zval(meta, "error.stack", &output);
 }
 
-static void _serialize_exception(zval *el, zval *meta, zend_object *exception_obj) {
-    zval exception, name, msg, stack;
+static void dd_serialize_exception(zval *el, zval *meta, zend_object *exception_obj) {
+    zval exception, name, msg, code, stack;
```

Comment:
Should we perhaps initialize `code` to undef? Or perhaps all of the zval's that are passed into `zend_call_method_with_0_params` just in case? :)

Reply:
On one hand this gives us better safety if something does go wrong, but if it's unnecessary then tools like ASAN are less helpful if you initialize it to something like IS_UNDEF.

I'll look into `zend_call_method_with_0_params` and company to see if we're outright having bugs, of course.

Reply:
For posterity: on PHP 7 `zend_call_function` sets the retval to IS_UNDEF; `zend_call_method` doesn't do anything with it prior. This means we probably _shouldn't_ initialize it, but it's difficult to know when we should or shouldn't initialize a variable. Maybe we can look at linters that support custom rules to help us get it right.